onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -329,6 +325,24 @@ ORT_API_STATUS_IMPL(OrtApis::AddCustomOpDomain, _In_ OrtSessionOptions* options,
   API_IMPL_END
 }
 
+ORT_API_STATUS_IMPL(OrtApis::RegisterCustomOpsLibrary, _Inout_ OrtSessionOptions* options, _In_ const char* library_path) {
+  API_IMPL_BEGIN
+
+  void* handle;
+  Env::Default().LoadDynamicLibrary(library_path, &handle);
```

Comment:
What's our policy around unloading? Do we also take responsibility to unload the library on the users behalf? Or give them a handle back to unload?

Reply:
I added a final parameter 'library_handle' that returns the handle, and documented when to free it.